keyboards/db/db63/db63.h

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Copyright 2017 Sebastian Kaim <sebb@sebb767.de>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+#define _x_ KC_NO
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ * MATRIX_ROW_PINS { B0, B1, B2, B3, B4, B5 }
+ * MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6, C5, C4, C3, C2 }
+ *
+ */
+
+#define LAYOUT_63_ansi( \
```

Comment:
This board seems to only supports 1 non community layout
```suggestion
#define LAYOUT( \
```